client/jetpack-cloud/sections/partner-portal/license-lightbox/single-site-upsell-lightbox.tsx

```diff
@@ -33,21 +30,19 @@ export default function SingleSellUpsellLightbox( {
 
 	const { hideLicenseInfo } = useContext( SitesOverviewContext );
 
-	const item = slugToSelectorProduct( productSlug ) as SelectorProduct;
-
-	const selectedSiteSlug = useSelector( ( state ) => getSiteSlug( state, siteId ) ) || '';
-
-	const createCheckoutURL = getPurchaseURLCallback( selectedSiteSlug, {
-		// For the Backup, Search, Scan upsell in Jetpack Cloud, we want to redirect back here to the Backup page after checkout.
-		redirect_to: window.location.href,
-	} );
-
-	const checkoutURL = createCheckoutURL && createCheckoutURL( item, false );
 	const onHideLicenseInfo = useCallback( () => {
 		hideLicenseInfo();
 		onClose?.();
 	}, [ hideLicenseInfo, onClose ] );
 
+	const sites = useSelector( getSites );
+
+	const selectedSite = siteId
+		? sites.find( ( site ) => site?.ID === parseInt( siteId as unknown as string ) )
+		: null;
+
+	const { submitForm } = useSubmitForm( selectedSite );
```

Comment:
What is this used for?

Reply:
This is the logic I've used to make sure the Issue License button redirects to the payment card page, following the same logic used on the `partner-portal/licenses` page itself (at `client/jetpack-cloud/sections/partner-portal/issue-license-v2/review-licenses/pricing-summary.tsx`)